Documentation/Classes.txt

```diff
@@ -1,20 +1,78 @@
 Game Classes:
+
   => Game
-  => Galaxy
-    - struct Planet
-      > enum Status
+    - Fields
+      > Galaxy
+    - Methods
+      > Handle Events
+      > Update
+      > Render
+
+  => Galaxy (maybe pointless?)
```

Comment:
lmao in all honesty, it's only purpose is to make the planet array, and we pass that into map, so why not just have map do it all?

Reply:
You do not want Map to contain the planets. It has nothing to do with anything but UI and rendering. We may end up just instantiating a planets array called galaxy in Game, but it is also possible we will replace the galaxy class with a GalaxyManager that constructed by passing an array of planets and the galaxy manager then has methods to handle events and update the planets by calling planet methods i.e. getters and setters for deposits, minerals, population